packages/web-components/src/components/cbp-list/cbp-list.scss

```diff
@@ -108,22 +106,26 @@ cbp-list {
 }
 
   //Link List
+  &[variant="link"]{
+    padding-inline-start: var(--cbp-space-8x);
+    
+    li{
+      list-style-type: none;
+  
+      .linkListIcon, 
```

Comment:
I would simply target `cbp-icon` here. Forcing the end-user/developer to use a specific class name for slotted content isn't ideal. They may get the code from storybook, but otherwise there's no indication what classes they are supposed to use. This is a pretty simple use case and they shouldn't be sprinkling random icons within their lists otherwise.